cmd/expose.go

```diff
@@ -117,7 +126,7 @@ ktunnel expose redis 6379
 				if err != nil {
 					log.Fatalf("Failed to run client: %v", err)
 				}
-			}()
+			}(srcPort)
```

Comment:
what's the benefit of passing the port to the goroutine?

Reply:
https://github.com/golang/go/wiki/CommonMistakes#using-goroutines-on-loop-iterator-variables